### PR TITLE
New secrets CLI and updated secrets docs

### LINKED ISSUE #750: Document 'influx secret' command and subcommands
You can now manage secrets using the `influx secret` command and subcommands.

PR: https://github.com/influxdata/influxdb/pull/16786

##### Relevant URLs
- [x] https://v2.docs.influxdata.com/v2.0/reference/cli/influx/
- [x] https://v2.docs.influxdata.com/v2.0/security/secrets/manage-secrets/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #750

This adds the new `influx secret` CLI with its subcommands. It also extends the secret management docs to include instructions for managing secrets with the CLI and API.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/cli/influx/secret/_index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+weight: 101
+v2.0/tags: [secrets]
+---
+
+The `influx secret` command manages secrets.
+
+## Usage
+```
+influx secret [flags]
+influx secret [command]
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/secret/_index.md]: should we say influx secret [subcommand]? 

or how about combining flags/commands to options?:
influx secret [options]
## Options
delete
find
update
help

File: content/v2.0/reference/cli/influx/secret/_index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+## Subcommands
+| Subcommand                                          | Description            |
+|:----------                                          |:-----------            |
+| [delete](/v2.0/reference/cli/influx/secret/delete/) | Delete a secret        |
+| [find](/v2.0/reference/cli/influx/secret/find/)     | Find secrets           |
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/secret/_index.md]: List secrets or Find a secret?

File: content/v2.0/reference/cli/influx/secret/delete.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+weight: 101
+v2.0/tags: [secrets]
+---
+
+The `influx secret delete` command deletes secrets.
+
+## Usage
+```
+influx secret delete [flags]
+influx secret delete [command]
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/secret/delete.md]: think we can drop "influx secret delete [command] here? 
then use the same term "flags" or "options" across the board?

File: content/v2.0/reference/cli/influx/secret/delete.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+```
+
+## Flags
+| Flag           | Description                 | Input type |
+|:----           |:-----------                 |:----------:|
+| `-h`, `--help` | Help for `delete`           |            |
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/secret/delete.md]: Help for the `secret delete` command

File: content/v2.0/reference/cli/influx/secret/find.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+weight: 101
+v2.0/tags: [secrets]
+---
+
+The `influx secret find` command lists secret keys.
+
+## Usage
+```
+influx secret find [flags]
+influx secret find [command]
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/secret/find.md]: options?

File: content/v2.0/security/secrets/manage-secrets/_index.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+title: Manage secrets
+description: Manage secrets in InfluxDB with the InfluxDB API.
+v2.0/tags: [secrets, security]
+menu:
+  v2_0:
+    parent: Store and use secrets
+weight: 201
+---
+
```
Comment (kelseiv) [inline content/v2.0/security/secrets/manage-secrets/_index.md]: extra line

File: content/v2.0/security/secrets/manage-secrets/delete.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+## Delete secrets using the InfluxDB API
+Use the `POST` request method and the `orgs/{orgID}/secrets/delete` API endpoint
+to delete one or more secrets.
+
+**Include the following:**
+
+- Your [organization ID](/v2.0/organizations/view-orgs/#view-your-organization-id) in the request URL
+- Your [authentication token](/v2.0/security/tokens/view-tokens/) in the `Authorization` header
+- An array of secret keys to delete in the requests body
```
Comment (kelseiv) [inline content/v2.0/security/secrets/manage-secrets/delete.md]: request body

File: content/v2.0/security/secrets/manage-secrets/update.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    parent: Manage secrets
+weight: 303
+---
+
+Update secrets using the `influx` command line interface (CLI) or the InfluxDB API.
+
+## Update a secret using the influx CLI
+Use the [`influx secret update` command](/v2.0/reference/cli/influx/secret/update/)
+to update a secret in your organization.
+Provide the secret key with the the `-k` or `--key` flag.
```
Comment (kelseiv) [inline content/v2.0/security/secrets/manage-secrets/update.md]: extra the

File: content/v2.0/reference/cli/influx/secret/_index.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/secret/_index.md]: The current code block copies the output of the actual CLI help, although I'm fine with updating it to "subcommand." There is a distinct difference between commands/subcommands and flags. Commands and subcommands perform an action; flags modify that action in some way. In the context of CLIs, it's important to distinguish the two.

File: content/v2.0/reference/cli/influx/secret/_index.md (hunk 2 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/secret/_index.md]: Good catch. The tools team is actually changing `find` to `list` across all CLI commands. My plan is to update this in another PR when that change is merged. See https://github.com/influxdata/docs-v2/issues/753.

File: content/v2.0/reference/cli/influx/secret/delete.md (hunk 3 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/secret/delete.md]: See previous comment

Comment (kelseiv) [reply content/v2.0/reference/cli/influx/secret/delete.md]: If delete is the only command, think we can drop [command]?
...reads like it's a command with other sub-commands available, like `secret` 
think in other places we say?:
## Usage
influx secret delete
influx secret delete [flags]

File: content/v2.0/reference/cli/influx/secret/delete.md (hunk 4 above)
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/secret/delete.md]: @sanderson, Help for the `secret delete` command...or Help for `secret delete`? -update across the board for other commands

File: content/v2.0/reference/cli/influx/secret/update.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+menu:
+  v2_0_ref:
+    name: influx secret update
+    parent: influx secret
+weight: 101
+v2.0/tags: [secrets]
+---
+
+The `influx secret update` command adds and updates secrets.
+Provide the secret key with the the `-k` or `--key` flag.
```
Comment (kelseiv) [inline content/v2.0/reference/cli/influx/secret/update.md]: @sanderson there's an extra "the" here

File: content/v2.0/reference/cli/influx/secret/delete.md (hunk 3 above)
Comment (sanderson) [reply content/v2.0/reference/cli/influx/secret/delete.md]: You're right. I noticed the context and made this update this morning.

**Pull request merged**